README.md

```diff
@@ -78,7 +79,6 @@ cp vars.local.sh vars.sh
 
 ### Using the helper script for Docker Compose
 ```bash
-cd docker/
```

Comment:
Why is this removed?

Reply:
I asked Elliot to make this change.
As when you follow the read me you're already in the docker directory from step before (when copying the vars shell scripts).
Just like in the GP2GP readme.